dist/eol-tracker.spec

```diff
@@ -21,6 +21,14 @@ BuildRequires:  %{_bindir}/yarnpkg
 
 Requires:       distribution-gpg-keys
 
+# plugins that aren't found during build
+Requires:       cutelyst2-plugins
+Requires:       qt5-qtbase-postgresql
+
+# translation files
+Requires:       libCutelyst2Qt5-2-lang
+Requires:       libCutelyst2Qt5ViewGrantlee2-lang
```

Comment:
What warning were you seeing that warranted this?

Reply:
https://github.com/cutelyst/cutelyst/blob/f66666bf68a8666bcb8c57447bc1ac3a28b70d6f/Cutelyst/application.cpp#L555 and https://github.com/cutelyst/cutelyst/blob/416a70a4b2945a83e5ca61dbad7e6e2e372f2ba3/Cutelyst/Plugins/View/Grantlee/grantleeview.cpp#L314

Reply:
These are bugs in the cutelyst packaging you're using, and I'd rather you didn't include this here...